person.rb

```diff
@@ -1,17 +1,18 @@
 require './corrector'
 
 class Person
+  attr_accessor :name, :age
+  attr_reader :rentals, :id
+
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new(@name)
+    @rentals = []
   end
 
-  attr_accessor :name, :age
-  attr_reader :id
-
   def of_age?
     true if @age >= 18
     false
```

Comment:
- [ ] We are missing the method for adding rental after passing (book and date)

Reply:
- [ ] Please address this

Reply:
## * Hi, kindly address this issue pointed out by the previous reviewer as this would affect future milestones of this project🙏  
## * As you have created a method for adding rentals in `book.rb` which has an association of `has-many-rentals` you can do same for `person.rb` as it also have the same association of `has-many-rentals` 
![image](https://user-images.githubusercontent.com/52497246/137157190-12a474ba-b9a4-480f-b36a-325844805392.png)
## [Refer to this article for a refresher on associations](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)